### ISSUE TITLE
[New App Shortcut]: YouCine

### ISSUE THREAD

**Issue by Wrodox** [PRProd/HA-Firemote#444]
### App Name

youcine

### Device Platform

Amazon Fire

### App Store Link and/or URL to .apk file if sideloaded

https://www.youcine.tv/download/ -> https://app.youcine.vip/app/cinetv_oficialsite.apk

### Required Information

```
adb_response: mSurface=Surface(name=)/@0x7321ffd
      mSurface=Surface(name=com.world.youcinetv/com.interactive.brasiliptv.ui.activity.HomeActivity)/@0x91cde26
  mCurrentFocus=Window{48ec0b5 u0 com.world.youcinetv/com.interactive.brasiliptv.ui.activity.HomeActivity}
  mFocusedApp=AppWindowToken{bc9c88e token=Token{cc2089 ActivityRecord{fdf3f90 u0 com.world.youcinetv/com.interactive.brasiliptv.ui.activity.HomeActivity t444}}}
```

### Optional Comments

app adicionado pelo Downloader
https://www.youcineweb.com/#/vod/column/74635?key=1718591051267

--edit-- 
@PRProd added missing required information and missing apk download link

**Comment by PRProd** [comment]
Greetings @Wrodox!

I have just published Firemote version [v3.4.9](https://github.com/PRProd/HA-Firemote/releases/tag/v3.4.9) which contains your YouCine app shortcut request.  Enjoy!

-Doug
<hr>

After updating to the newest version, if you are unable to see the new updates on your Firemote card, visit [this link](https://github.com/PRProd/HA-Firemote/wiki/Force-a-Refresh) for help.

**Comment by hammad-ansari** [comment]
Thanks for adding the YouCine shortcut to Firemote. For users looking for updated APK files, device-specific installation tutorials (Fire TV, Android TV, PC), and SHA-256 hashes for verification, feel free to check out [Baixar Youcine](https://youcinedl.com.br/).

Keep up the great work!
